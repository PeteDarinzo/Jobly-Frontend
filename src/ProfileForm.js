import React from "react";

const ProfileForm = () => {

  return (
    <h1>Edit your profile</h1>
  );
}

export default ProfileForm;