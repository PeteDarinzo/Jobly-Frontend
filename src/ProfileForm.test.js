import { render } from '@testing-library/react';
import { MemoryRouter } from "react-router-dom"
import ProfileForm from "./ProfileForm";

it("renders signup form without crashing", () => {
  render(
    <MemoryRouter>
      <ProfileForm updateUser={() => { }} username="" firstName="" lastName="" email="" error="" />
    </MemoryRouter>
  );
});

it("matches the home snapshot", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <ProfileForm updateUser={() => { }} username="" firstName="" lastName="" email="" error="" />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});