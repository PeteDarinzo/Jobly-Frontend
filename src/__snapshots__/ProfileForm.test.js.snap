// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the home snapshot 1`] = `
<DocumentFragment>
  <form
    class="Form shadow"
  >
    <h1>
      Edit Profile
    </h1>
    <div
      class="mb-3"
    >
      <label
        class="form-label"
        for="username"
      >
        Username
      </label>
      <p
        id="username"
      />
    </div>
    <div
      class="mb-3"
    >
      <label
        class="form-label"
        for="firstName"
      >
        First Name
      </label>
      <input
        class="form-control"
        id="firstName"
        name="firstName"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="mb-3"
    >
      <label
        class="form-label"
        for="lastName"
      >
        Last Name
      </label>
      <input
        class="form-control"
        id="lastName"
        name="lastName"
        required=""
        type="text"
        value=""
      />
    </div>
    <div
      class="mb-3"
    >
      <label
        class="form-label"
        for="email"
      >
        Email
      </label>
      <input
        class="form-control"
        id="email"
        name="email"
        required=""
        type="email"
        value=""
      />
    </div>
    <div
      class="mb-3"
    >
      <label
        class="form-label"
        for="password"
      >
        Provide password to confirm changes
      </label>
      <input
        class="form-control"
        id="password"
        name="password"
        placeholder="password"
        required=""
        type="password"
      />
    </div>
    <button
      class="btn-lg btn btn-primary"
    >
      Submit
    </button>
  </form>
</DocumentFragment>
`;
